packages/backend/migrations/20241125224212_create_tenants_table.js

```diff
@@ -8,6 +8,9 @@ exports.up = function (knex) {
     table.string('email').notNullable()
     table.string('publicName')
     table.string('apiSecret')
+
+    table.timestamp('createdAt').defaultTo(knex.fn.now())
+    table.timestamp('updatedAt').defaultTo(knex.fn.now())
```

Comment:
Maybe also deletedAt ? So we would leverage soft delete. What do you think?

Reply:
Makes sense to me